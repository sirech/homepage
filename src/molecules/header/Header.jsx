import React from 'react'
import { Helmet } from 'react-helmet'

import { LinkContainer } from 'react-router-bootstrap'
import { Navbar, NavbarBrand } from 'reactstrap'

const Header = () => (
  <header className='mb-4'>
    <Helmet>
      <title>hceris</title>
    </Helmet>
    <Navbar color='faded'>
      <LinkContainer to='/about'>
        <NavbarBrand>Mario Fernandez</NavbarBrand>
      </LinkContainer>
    </Navbar>
  </header>
)

export default Header
