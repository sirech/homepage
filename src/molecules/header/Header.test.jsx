import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Header from './Header'

describe('components', () => {
  describe('Header', () => {
    it('renders correctly', () => {
      const component = renderer.create(
        <MemoryRouter>
          <Header />
        </MemoryRouter>)
      expect(component.toJSON()).toMatchSnapshot()
    })
  })
})
