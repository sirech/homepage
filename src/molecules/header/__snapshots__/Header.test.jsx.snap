// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Header renders correctly 1`] = `
<header
  className="mb-4"
>
  <nav
    className="navbar navbar-light bg-light"
  >
    <a
      className="navbar-brand"
      href="/about"
      onClick={[Function]}
      style={undefined}
    >
      Mario Fernandez
    </a>
  </nav>
</header>
`;
